database/youtube/data-structures.json

```
@@ -62,5 +62,13 @@
     "category": "youtube",
     "subcategory": "dsa",
     "language": "english"
+  },
+  {
+    "name": "MIT OpenCourseWare",
+    "description": "Explanation of Algorithms concepts. The resources for Algorithm are limited but this course is one of the best Algorithm resource available on Web.",
```

[Proofreading] This description is a bit ambiguous. Consider using this question to help you answer it: What makes the source the best for learning about Algorithms? 
```suggestion
    "description": "Explanation of Algorithms concepts. The resources for Algorithm are limited but this course is one of the best Algorithm resource available on Web.",
```

---

Okay I will make changes according to your suggestions.